### ISSUE TITLE
Accessibility overview

### ISSUE THREAD

**Issue by hannahackermans** [elmcip/elmcip#489]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/hac061/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/hac061/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

I have tested ELMCIP for Web Content Accessibility Guidelines (WCAG) 2.1 International Standard according to the requirements of uutilsynet.no, which covers all A and AA success criteria, but not AAA. As I said in the report as well, this is my first WCAG assessment and I learned a lot and I expect to continue learning throughout this process. So there might be some things I’ve overlooked so far that I will address later on. 

The table gives an overview of the current state of the accessibility of ELMCIP. I will turn the failures into GitHub issues labeled with "accessibility" and I will update this table to include the right issues. There are a number of WCAG failures that we cannot fix. I will list them here as "fail, but exempt", but I will not create issues for them as this point. 

WCAG requirement  | current status
-- | --
1.1.1 Non-text Content | fail, but exempt
  |  
1.2.1   Audio-only and Video-only (Prerecorded) | fail, but exempt
1.2.2 Captions   (Prerecorded) | fail, but exempt
  |  
1.3.1 Info and  Relationships | fail, variety of issues that I will get into later
1.3.2 Meaningful   Sequence | pass
1.3.3 Sensory   Characteristics | fail, issue #491
1.3.4 Orientation | pass
1.3.5 Identify   Input Purpose | pass, but potential for improvement
  |  
1.4.1 Use of Color | fail, issue #491
1.4.2 Audio Control | not applicable
1.4.3 Contrast   (Minimum) | fail, issue #492
1.4.4 Resize text | fail, issue #490 
1.4.5 Images of Text | fail, but exempt
1.4.10 Reflow | fail, issue #490 
1.4.11 Non-text   Contrast | fail, issue #492
1.4.12 Text Spacing | fail, issue #490 
1.4.13 Content   on Hover or Focus | not applicable
  |  
2.1.1 Keyboard | pass
2.1.2 No Keyboard Trap | not applicable
2.1.4 Character   Key Shortcuts | not applicable
  |  
2.2.1 Timing   Adjustable | not applicable
2.2.2 Pause, Stop, Hide | not applicable
  |  
2.3.1   Three Flashes or Below Threshold | fail, but exempt
  |  
2.4.1 Bypass Blocks | fail, issue #498
2.4.2 Page Titled | pass
2.4.3 Focus Order | pass
2.4.4 Link   Purpose (In Context) | fail, issue #494 
2.4.5 Multiple Ways | pass
2.4.6 Headings and   Labels | pass
2.4.7 Focus Visible | fail, issue #493
  |  
2.5.1 Pointer Gestures | not applicable
2.5.2 Pointer   Cancellation | pass
2.5.3 Label in Name | marked as fail but need to discuss with @steinmb 
2.5.4 Motion Actuation | not applicable
  |  
3.1.1 Language of Page | fail, issue #495 
3.1.2 Language of   Parts | fail, issue #495 
  |  
3.2.1 On Focus | pass
3.2.2 On Input | pass
3.2.3 Consistent   Navigation | pass
3.2.4   Consistent Identification | fail, issue #496
  |  
3.3.1 Error   Identification | fail, issue #497 
3.3.2 Labels or   Instructions | pass
3.3.3 Error Suggestion | fail, issue #497 
3.3.4   Error Prevention (Legal, Financial, Data) | not applicable
  |  
4.1.1 Parsing | marked as fail but need to discuss with @steinmb 
4.1.2 Name, Role, Value | marked as fail but need to discuss with @steinmb 
4.1.3 Status Messages | fail, issue ####

</body>

</html>

**Comment by steinmb** [comment]
Fantastic work!
